templates/kubernetes/what-is-kubernetes.html

```diff
@@ -143,12 +185,88 @@ <h2 class="u-sv3">Kubernetes advantages</h2>
   </div>
 </section>
 
-<section class="p-strip--light">
+<section class="p-strip--suru">
+  <div class="u-fixed-width">
+    <h2>Kubernetes resources</h2>
+    <div class="row p-divider u-no-padding--left u-no-padding--right is-dark u-equal-height">
+      <div class="col-3 p-divider__block">
+        <div class="p-heading-icon__header">
+          {{
+            image(
+            url="https://assets.ubuntu.com/v1/30037eac-datasheet-white.svg",
+            alt="",
+            width="32",
+            height="28",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
+            ) | safe
+          }}
+          <h3 class="p-heading--4">Datasheet</h3>
+        </div>
+        <p><a class="p-link--inverted" href="https://assets.ubuntu.com/v1/b5f9ae49-Enterprise_Kubernetes_Datasheet.pdf">Kubernetes for the enterprise</a></p>
```

Comment:
It looks good now but would it be better if it's wrapped by `p-heading-icon--small` or `p-heading-icon`. Feel free to update it if that's better you think! 

```suggestion
     <div class="p-heading-icon--small">
        <div class="p-heading-icon__header">
          {{
            image(
            url="https://assets.ubuntu.com/v1/30037eac-datasheet-white.svg",
            alt="",
            width="32",
            height="28",
            hi_def=True,
            loading="lazy",
            attrs={"class": "p-heading-icon__img"},
            ) | safe
          }}
          <h3 class="p-heading--4">Datasheet</h3>
        </div>
        <p><a class="p-link--inverted" href="https://assets.ubuntu.com/v1/b5f9ae49-Enterprise_Kubernetes_Datasheet.pdf">Kubernetes for the enterprise</a></p>
      </div>
```

Reply:
I don't think it will make a difference visually but can add it in anyway